defaults/main.yml

```diff
@@ -1,4 +1,6 @@
 ---
+jenkins_connection_delay: 5
+jenkins_connection_retries: 12
```

Comment:
Let's go back to 300 seconds (so `60` for the default here), just to be generous. The first Jenkins startup can take a while.